jupyterlab_gcedetails/jupyterlab_gcedetails/handlers.py

```diff
@@ -30,11 +30,9 @@
     'http://metadata.google.internal') + '/computeMetadata/v1/?recursive=true'
 METADATA_HEADER = {'Metadata-Flavor': 'Google'}
 MACHINE_TYPE_CMD = 'gcloud compute machine-types describe'
```

Comment:
Not in scope for this change, but do we still need to use the machine types describe call?